### PR TITLE
Add documentation for Mars on Ray

### PR DESCRIPTION
**Pull request by hekaisheng**
## Why are these changes needed?

Add doc about how to running Mars jobs on Ray.

## Checks

- [ ] I've run `scripts/format.sh` to lint the changes in this PR.
- [ ] I've included any doc changes needed for https://docs.ray.io/en/latest/.
- [ ] I've made sure the tests are passing. Note that there might be a few flaky tests, see the recent failure rates at https://ray-travis-tracker.herokuapp.com/.
- Testing Strategy
   - [ ] Unit tests
   - [ ] Release tests
   - [ ] This PR is not tested (please justify below)

### CODE REVIEW

**Review by ericl — APPROVED** [review]

**Comment by ericl** [comment]
Looks good, pending travis build passing.

File: doc/source/mars_on_ray.rst [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    import mars.tensor as mt
+
+    t = mt.random.rand(100, 4, chunk_size=30)
+    df = md.DataFrame(t, columns=list('abcd'))
+    print(df.describe().execute())
+
+
+.. warning::
+
+  Mars remote API is not available for now.
```
Comment (stephanie-wang) [inline doc/source/mars_on_ray.rst:53]: What is the "remote API"?

File: doc/source/index.rst [hunk 2]
```diff
@@ -211,6 +211,7 @@ Papers
    iter.rst
    pandas_on_ray.rst
    dask-on-ray.rst
+   mars_on_ray.rst
```
Comment (anabranch) [inline doc/source/index.rst:214]: can you use dashes instead of underscores?

File: doc/source/mars_on_ray.rst (hunk 1 above)
Comment (anabranch) [reply doc/source/mars_on_ray.rst:53]: Yes, please clarify !

**Review by bllchmbrs — COMMENTED** [review]
Couple of review comments.

File: doc/source/index.rst (hunk 2 above)
Comment (hekaisheng) [reply doc/source/index.rst:214]: Fixed

File: doc/source/mars_on_ray.rst (hunk 1 above)
Comment (hekaisheng) [reply doc/source/mars_on_ray.rst]: I have added a hyperlink for remote API.

**Comment by hekaisheng** [comment]
@ericl hi, could you take a look and get this PR merged?

**Comment by ericl** [comment]
LINT is still failing

**Comment by hekaisheng** [comment]
> LINT is still failing

Sorry, I missed the failure, travis is green now.

**Comment by stephanie-wang** [comment]
Thanks, @hekaisheng!

**Pull request merged**